Source/DafnyPipeline.Test/PluginsTest.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Xunit;
+using Microsoft.Dafny;
+
+namespace DafnyPipeline.Test;
+
+[Collection("Dafny plug-ins tests")]
+public class PluginsTest {
+  /// <summary>
+  /// This method creates a library and returns the path to that library.
+  /// The library extends a Rewriter so that we can verify that Dafny invokes it if provided in argument.
+  /// </summary>
+  public async Task<string> GetLibrary(string code) {
```

Comment:
Since this is a different test project we may want to double think that. However, I'd suggest to share these common test parts between the compiler and the language server.

Reply:
I'd suggest to put these test parts in a common file too. However, I don't think it's relevant to put it in DafnyPipeline, and I don't think we should have DafnyLanguageServer.Test depend on DafnyPipeline.Test nor the other way around.
So the only solution would be to create another project, but if it just for the sake of putting this single method, it might be overkill.